src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="field-forms">
+          Surname:
+          <input
+          autocomplete="off"
+          type="text"
+          id="surname"
+          minlength="5"
+          maxlength="16"
+          required
+          >
+        </label>
+
+        <label for="name" class="field-forms">
+          Name:
+          <input
+          autocomplete="off"
+          type="text"
+          id="name"
+          >
+        </label>
+
+        <label for="age" class="field-forms">
+          How old are You?
+          <input
+          type="number"
+          id="age"
+          min="0"
+          max="100"
+          value="12"
+          >
```

Comment:
you need to add `name` attribute to ALL inputs
```suggestion
          <input
            type="number"
            id="age"
            min="0"
            max="100"
            value="12"
            name="age"
          >
```